README.md

```diff
@@ -28,3 +34,45 @@ Wekan is an open-source kanban board (task manager and organizer)
 **Private/Public mode:** In private mode, only authorized YunoHost members can access to the wekan. 
 
 **SSO/LDAP:** SSO and LDAP are not configured.
+
+## Configuration
+
+First registered user will be admin, and next ones normal users. If you want other admins too, you can change their permission to admin at Wekan Admin Panel.
+
+## Documentation
+
+ * Official documentation: https://github.com/wekan/wekan/wiki
+
+## YunoHost specific features
+
+#### Supported architectures
+
+* x86-64b - [![Build Status](https://ci-apps.yunohost.org/ci/logs/wekan%20%28Community%29.svg)](https://ci-apps.yunohost.org/ci/apps/wekan/)
+* ARMv8-A - [![Build Status](https://ci-apps-arm.yunohost.org/ci/logs/wekan%20%28Community%29.svg)](https://ci-apps-arm.yunohost.org/ci/apps/wekan/)
+* Jessie x86-64b - [![Build Status](https://ci-stretch.nohost.me/ci/logs/wekan%20%28Community%29.svg)](https://ci-stretch.nohost.me/ci/apps/wekan/)
```

Comment:
Uuuuh, I think I would remove those last two, or add a note : 
- This app doesn't support ARM
- We are in 2019 and we really have other things to do that trying to still support Jessie when Buster is coming in like 4 months :/

Reply:
For ARM, it seems that Wekan can be installed on ARM: https://blog.wekan.team/2018/01/wekan-progress-on-x64-and-arm/index.html => ARM platforms status 2018-01
For Jessie, I agree with you but maybe some users are still on Jessie and this line is in the example_ynh... I propose to remove it when it will be removed from example_ynh ;)

Reply:
For ARM, I remember now that you don't use directly the sources from Wekan project but do a specific build... And the build is only done for Stretch...

Reply:
Uh wait I think there's a confusion between the debian version (stretch vs. jessie) and arch (e.g. x86 vs. ARM)

For ARM there clearly isn't any support (though yes, some people in the Wekan team seem to have been working on ARM, but even then that's for ARM64 and not ARMHF (basically ARM 32 bit or something like that) which is another tricky thing.

For Jessie, I don't know if the build we use is compatible with Stretch, but anyway the app manifest requires YunoHost >= 3.x (because of helpers and other things) so Jessie support is just irrelevant :/

Reply:
Seem that wekan_ynh is supported for jessie: https://ci-stretch.nohost.me/ci/job/873 from 14/02/2019